File: Challenges/Coding World Cup/2023/Day 3 - SA vs SL - Chocolate Removal.cpp
long long totalChocolates(int n, vector<int> a)
{
    // Write your code here.
    long long ans = 0;
    for (int i = 0; i < n; i++)
    {
        if (a[i] > 23)
            ans += a[i] - 23;
    }
    return ans;
}

File: Challenges/Coding World Cup/2023/Day 3 - SA vs SL - Chocolate Removal.java
public class Solution {
    static long totalChocolates(int n, int[] a) {
        // Write your code here.
        int answer = 0;
        if (n == 0) {
            return 0;
        }
        for (int i = 0; i < n; i++) {
            if (a[i] > 23) {
                answer += (a[i] - 23);
            }
        }
        return answer;
    }
}